content/kotlin/concepts/math-methods/terms/ceil/ceil.md

```diff
@@ -0,0 +1,19 @@
+---
+**Title:** `.ceil()`
+
+**Description:** "The `.ceil()` method in Kotlin is used to round a floating-point number up to the nearest integer."
+
+**Syntax:** 
+  ```kotlin
+  kotlin.math.ceil(x: Double)
```

Comment:
```suggestion
kotlin.math.ceil(myDouble)
```

Reply:
does this only accept doubles?

Reply:
The kotlin.math.ceil() function in Kotlin accepts any numeric type that can be implicitly or explicitly converted to a Double. This means it can accept not only Double but also other numeric types like Float, Int, Long, and so on.

I will update the example accordingly.